adapters/unicorn/unicorn_test.go

```diff
@@ -0,0 +1,20 @@
+package unicorn
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderUnicorn, config.Adapter{
+		Endpoint: "http://localhost:4000"})
```

Comment:
This doesn't need to be a real endpoint. Why are you specifying a port?

Reply:
There is no reason. Changed to `https://ds.uncn.jp` for now.